_data/chains/eip155-31612.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Mezo",
+  "chain": "Mezo",
```

Comment:
Duplicated chain name between mainnet and testnet.

Also seemingly missing `icon`?

Reply:
Fixing now.